views/scripts/fixture-editor.js

```diff
@@ -99,6 +98,16 @@ Vue.component('fixture-mode', {
     addChannel: function() {
       this.channel.modeId = this.mode.uuid;
       this.channel.editMode = 'add-existing';
+    },
+    isChannelNameAmbigous: function(chKey) {
```

Comment:
Should rather be a `computed` property instead of a `method`. Also, have a look [here](http://www.dictionary.com/browse/ambiguous) 😉